packages/atomic/src/components/commerce/atomic-commerce-no-products/e2e/page-object.ts

```diff
@@ -21,7 +21,7 @@ export class NoProductsPageObject extends BasePageObject<'atomic-commerce-no-pro
   message(query?: string) {
     return this.page.locator('[part="no-results"]', {
       hasText: query
-        ? `We couldn't find any product for “${query}”`
+        ? `We couldn't find any product for ${query}}"`
```

Comment:
You sure about this ? This single double quote is suspicious

Reply:
Fixed 😄